source/basic.tex

```diff
@@ -1,8 +1,9 @@
 %!TEX root = std.tex
 \rSec0[basic]{Basic concepts}
 
-%gram: \rSec1[gram.basic]{Basic concepts}
-%gram:
+\begin{extract}
```

Comment:
Do you mean something integrated like `\grammarheading{gram.basic}{Basic concepts}`?